### PR TITLE
[hw-spring-boot-data-jpa] implemented update and delete logic

### PR DESCRIPTION
**Pull request by Andriy-Hrynevetskyi**

### CODE REVIEW

File: src/main/resources/db/changelog/db.changelog-master.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+databaseChangeLog:
+  - include:
+      file: db/changelog/changes/01-create-books-table.yaml
```
Comment (MinoDentori) [inline src/main/resources/db/changelog/db.changelog-master.yaml]: Add a new line to the end of the file)

Screenshot 2023-12-08 at 20 01 23

**Review by MinoDentori — APPROVED** [review]
Good job! Just one small moment to proceed.

File: src/main/resources/db/changelog/db.changelog-master.yaml (hunk 1 above)
Comment (Andriy-Hrynevetskyi) [reply src/main/resources/db/changelog/db.changelog-master.yaml]: ok ty)

File: src/main/java/bookstore/controller/BookController.java [hunk 2]
```diff
@@ -33,4 +37,17 @@ public BookDto getBookById(@PathVariable Long id) {
     public BookDto createBook(@RequestBody CreateBookRequestDto requestDto) {
         return bookService.save(requestDto);
     }
+
+    @ResponseStatus(HttpStatus.NO_CONTENT)
```
Comment (d1sam) [inline src/main/java/bookstore/controller/BookController.java]:
```suggestion
```

File: src/main/resources/db/changelog/changes/01-create-books-table.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-books-table
+      author: andriy
```
Comment (d1sam) [inline src/main/resources/db/changelog/changes/01-create-books-table.yaml]: you should also write your lastname

**Review by d1sam — CHANGES_REQUESTED** [review]
Good job, but some changes should be done!

**Review by d1sam — APPROVED** [review]
Well done!

**Pull request merged**